### PR TITLE
[VM] Make UIButton extension positionSubviews overridable

### PR DESCRIPTION
**Pull request by charleyxiv**
## Description
UIButton subclasses can now override `positionSubviews` to make further adjustments once layout was adjusted to respect the image's alignment.
 
## How Has This Been Tested?

I tested that a subclass of `UIButton` could override `positionSubviews`:
```
private class Test: UIButton {

    override func positionSubviews(_ alignment: Alignment?) {
        super.positionSubviews(alignment)
    }
}
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by marcantoinefortier — APPROVED** [review]

File: trikot-viewmodels/swift-extensions/UIButtonExtensions.swift [hunk 1]
```diff
@@ -156,7 +156,8 @@ extension UIButton {
         return completedAttributedString
     }
 
-    public func positionSubviews(_ alignment: Alignment?) {
+    @objc
```
Comment (marcantoinefortier) [inline trikot-viewmodels/swift-extensions/UIButtonExtensions.swift:159]: Why is this required?

Comment (charleyxiv) [reply trikot-viewmodels/swift-extensions/UIButtonExtensions.swift:159]: I needed it to fix this error:

> Non-@objc instance method 'positionSubviews' is declared in extension of 'UIButton' and cannot be overridden

Comment (antoinelamy) [reply trikot-viewmodels/swift-extensions/UIButtonExtensions.swift:159]: Maybe a more elegant way of achieving the same result would have been to conditionally check for a specific protocol conformance instead of overriding a function from an extension.
